db/schema.rb

```diff
@@ -145,6 +145,7 @@
     t.text "feedback_suggestions"
     t.float "latitude"
     t.float "longitude"
+    t.boolean "feature_restructured_work_history", default: true
```

Comment:
Defaulting this to true means that all new application forms will by default use the new flow. New applications being redirected to the old flow prior to the flag being switched on is dealt with in a later commit.